main.cpp

```diff
@@ -227,6 +265,21 @@ struct AppleTree
     void growApples(int soilQuality = 7, bool sunny = true);
     void provideShade(double leaflessness = 95.5, bool sunny = true);
     void dropApples(int appleRipeness = 75);
+    float aging(float treeAge)
+    {
+        treeAge = age;
+        for(float i = treeAge; i < 66; i ++)
+        {
+            age = i;
+            if(age >= 65.f)
+            {
+                std::cout << "ok imma retire now" << std::endl;
+            }
+        }
+        treeAge = age;
```

Comment:
again, overwriting the function argument with your member variable's value.
- [ ] revise